server/README.md

```diff
@@ -93,15 +93,20 @@ At the moment there is a limit to how many updates the Dependabot will recommend
 You can use swagger to test the functionality of the currently available API's. 
 You can find more information on the site: https://swagger.io/solutions/api-documentation/
 
+For Back End Users:
+
 Run the following commands to seed the test database & start the local server: 
 
 ```
 rails db:seed
 bundle exec rails server
 ```
-
 Then visit the local swagger site http://localhost:3000/api-docs/index.html
 
+For Front End Users: 
+
+For those on the front end, swagger docs for the api can be found at https://mentor-mentee-platform.herokuapp.com/api-docs/index.html
```

Comment:
```suggestion
For those on the front end, consult the [published Swagger API docs](https://mentor-mentee-platform.herokuapp.com/api-docs/index.html).
```

Reply:
We have the full power of hyperlinks at our disposal! 🌟

Reply:
Come to think of it, perhaps the section for the frontend audience should go _first_, so that they can skip the rest of the detail.